### PR TITLE
About page

### PR DESCRIPTION
**Pull request by remy-rm**
création de la page About , 
-ajout du dossier model (contient le model class team) , 
-service(contient le dossier pour le service team) et 
-dossier style ( contient les fichier _variables.scss et _animations.scss )

### CODE REVIEW

**Review by td6904 — DISMISSED** [review]

File: src/app/pages/about/about.component.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<section class="container">
+    <h1>About The <span>Team</span></h1>
+    <p>
+        Chef’s Compass is a recipe finding app for you, the user. It helps you find a recipe based on your
+        hunger-tackling desires and any ingredients stocked in your kitchen. Not sure how to use them effectively? Let
+        Chef’s Compass show you the way.
+      </p>
+
+<div class="team">
+    <section class="team_member" *ngFor="let team of teams" >
```
Comment (Benja-san) [inline src/app/pages/about/about.component.html:10]: your ng for will generate a section for each member, put at the beginning of the member card.
Create an ul an li as well for better referencing

File: src/app/pages/about/about.component.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        <div class="team_member--image">
+            <img [src]="team.getImage()" [alt]="team.getName">
+        </div>
+        <div class="team_member--info">
+            <h2>{{team.getName()}}</h2>
+            <h4>Web Developer</h4>
```
Comment (Benja-san) [inline src/app/pages/about/about.component.html:16]: where is the h3 ?

**Review by Benja-san — CHANGES_REQUESTED** [review]
If you can change import for use in scss file it is better practice.

File: src/app/pages/about/about.component.html (hunk 2 above)
Comment (remy-rm) [reply src/app/pages/about/about.component.html:16]: i change in h3

**Review by td6904 — APPROVED** [review]

**Review by td6904 — APPROVED** [review]

File: src/app/app-routing.module.ts [hunk 3]
```diff
@@ -2,15 +2,19 @@ import { NgModule } from '@angular/core';
...
 import { AboutComponent } from './pages/about/about.component';
 import { HomepageComponent } from './homepage/homepage.component';
+import { DemoPageComponent } from './pages/demo/demo.component';
 
 const routes: Routes = 
 [
   {path: 'about', component: AboutComponent},
-  { path: '', component: HomepageComponent } 
+  { path: '', component: HomepageComponent }, 
+  { path: '', component: DemoPageComponent }
```
Comment (Benja-san) [inline src/app/app-routing.module.ts:11]: path '' leads to two components

File: src/app/pages/about/about.component.scss [hunk 4]
```diff
@@ -1,9 +1,14 @@
-@import '../../style/_variables.scss' ,'../../style/_animation.scss';
+@use '../../style/_variables' as *  ;
+@use '../../style/_animation'  as *;
 h1{
```
Comment (Benja-san) [inline src/app/pages/about/about.component.scss:3]: please use class with the parent component to wrap all the subjects inside it (can lead to css conflicts)

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by td6904 — APPROVED** [review]

**Review by Benja-san — APPROVED** [review]

**Pull request merged**